configure.ac

```diff
@@ -99,7 +99,7 @@ AX_PTHREAD()
 LIBS="$PTHREAD_LIBS $LIBS"
 CFLAGS="$PTHREAD_CFLAGS $CFLAGS"
 CC="$PTHREAD_CC"
-CXXFLAGS="$CXXFLAGS -ftemplate-depth=128 -Wno-format-zero-length"
+CXXFLAGS="$CXXFLAGS -ftemplate-depth=128 -Wno-format-zero-length -std=c++11"
```

Comment:
This is hardcoding GCC specifics. I was under the impression that there already are some C++11 detection logic in place in here, by auto conf